tools/common/Frameworks.cs

```diff
@@ -535,15 +535,8 @@ public static void Gather (Application app, AssemblyDefinition product_assembly,
 	// This checks if a framework is unavailable due to bugs in Xcode (such as Apple forgetting to ship a library or headers for a framework, which seems to happen at least once a year).
 	public static bool IsFrameworkBroken (Application app, string framework)
 	{
-		if (app.IsSimulatorBuild && Driver.XcodeProductVersion == "12A6163b" /* Xcode 12 beta 12 */) {
+		if (app.IsSimulatorBuild && Driver.XcodeProductVersion == "12A8169g" /* Xcode 12 beta 3 */) {
 			switch (framework) {
-			// Apple seems to have forgotten to ship the several libraries for the simulator in Xcode 12 betas (it's still available for device builds).
```

Comment:
This particular line is still correct for the MediaSetup framework, right?

Reply:
yes, I cut too large :)